pkg/service/operation/service.go

```diff
@@ -0,0 +1,45 @@
+package operation
+
+import (
+	"fmt"
+	sdkutil "github.com/TBD54566975/ssi-sdk/util"
+	"github.com/pkg/errors"
+	"github.com/tbd54566975/ssi-service/internal/util"
+	"github.com/tbd54566975/ssi-service/pkg/service/framework"
+	storage2 "github.com/tbd54566975/ssi-service/pkg/service/operation/storage"
```

Comment:
perhaps `opstorage` as an alias instead of `storage2`?

Reply:
Done.